breaking/action.yml

```diff
@@ -18,6 +18,14 @@ inputs:
     description: 'Include path parameter names in endpoint matching'
     required: false
     default: 'false'
+  deprecation-days-beta:
+    description: 'Consider minimum sunset period for deprecation of beta API endpoints'
+    required: false
+    default: '31'
+  deprecation-days-stable:
+    description: 'Consider minimum sunset period for deprecation of stable API endpoints'
+    required: false
+    default: '180'
```

Comment:
While the script checks if `deprecation_days_beta` and `deprecation-days-stable` are not empty, assigning a default value implies that these variables will always have a value, right?

Reply:
That's right. I removed the default values to fallback to the oasdiff defaults. Kept them in the README though 👍